website/docs/Best_practices/task_execution.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 6
+---
+
+# Task execution - tips for using the WDL task command section
+Every WDL task has a [command section](https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md#command-section) where you can call software tools and specify parameters to help transform your data into meaningful output. This section is like a terminal for whatever environment you’re using to execute your WDL script. That environment can be a virtual computer set up by a docker container or it can be your local computer. If you’re using Cromwell to execute your WDL (as happens in the cloud-based platform [Terra](app.terra.bio)), the command section is run after Cromwell has resolved the task inputs but before it assesses outputs. 
+
+The actual commands you use in the task command section depend on what tools and operating systems are available in the execution environment. [WARP](https://github.com/broadinstitute/warp/tree/master) pipelines, for example, often set up virtual machines using Docker containers with Alpine Linux-based operating systems. This means that the command section should contain commands that work in Alpine. If additional software is installed on top of Alpine, that software's commands should also work. WARP workflows often require custom python scripts and that’s why python is installed on top of WARP’s Alpine-based dockers. Python is one example, but you can install any language on a docker and then use language-specific commands or scripts from the WDL task command section.
+
+If a script is small, you can even write scripts in line. The [Resources](#resources) section below links to examples of workflow code that use in-line scripting. If you’re using in-line scripting, remember to keep your code tidy (check out the WARP WDL formatting suggestions). And remember, commands will only work if installed on the environment in which the WDL is running; what works on your local machine will not necessarily work in a virtual machine built off a docker and vice-versa.  
```

Comment:
perhaps this is obvious, but that can include things like where the software is (path to the jar for instance) and input/output file paths within the docker

Reply:
Thank you! Added this and modified slightly based on Tim's comments about Alpine not always being used as the option.